src/test/kotlin/uk/gov/dluhc/printapi/rest/GenerateTemporaryCertificateIntegrationTest.kt

```diff
@@ -31,6 +40,10 @@ internal class GenerateTemporaryCertificateIntegrationTest : IntegrationTest() {
             .uri(URI_TEMPLATE, ERO_ID)
             .bearerToken(UNAUTHORIZED_BEARER_TOKEN)
             .contentType(MediaType.APPLICATION_JSON)
+            .body(
+                Mono.just(buildGenerateTemporaryCertificateRequest()),
+                GenerateTemporaryCertificateRequest::class.java
+            )
```

Comment:
Minor suggestion, @NathanRussellValtech ,
We can have an extension function `withABody` to simplify this:
<pre>
        webTestClient.post()
            .uri(URI_TEMPLATE, ERO_ID)
            .bearerToken(getBearerToken(eroId = ERO_ID, groups = listOf("ero-$ERO_ID", "ero-admin-$ERO_ID")))
            .contentType(MediaType.APPLICATION_JSON)
<b>            .withABody(buildGenerateTemporaryCertificateRequest())</b>
</pre>
<pre>
    private fun <T : Any> WebTestClient.RequestBodySpec.withABody(requestBody: T): WebTestClient.RequestBodySpec =
        body(
            Mono.just(requestBody),
            requestBody.javaClass
        ) as WebTestClient.RequestBodySpec
</pre>

Reply:
Oh yes, I like that 👍 
I'll do that now 😁